deepmerge/strategy/type_conflict.py

```diff
@@ -15,3 +15,8 @@ def strategy_override(config, path, base, nxt):
     def strategy_use_existing(config, path, base, nxt):
         """ overrides the new object over the old object """
         return base
+
+    @staticmethod
+    def strategy_override_if_not_empty(config, path, base, nxt):
+        """ overrides the new object over the old object """
+        return nxt if nxt else base
```

Comment:
existence checks also return False for empty lists, dicts, and sets. Is this the intended behavior?

Otherwise an "if not null" might be good.

Reply:
Yes, this is exactly the intended behavior